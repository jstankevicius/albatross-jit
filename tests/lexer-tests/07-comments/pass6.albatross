number123 # comments 98badNumber
anotherNumber456 # not always 473badNumber
yetAnotherNumber789 # start on the same line @@@#(*&$(*&))(
