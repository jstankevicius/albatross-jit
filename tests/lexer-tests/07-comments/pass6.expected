1 1 NAME number123
1 2 NAME anotherNumber456
1 3 NAME yetAnotherNumber789
